tests/wpt/tests/css/css-text/white-space/break-spaces-with-word-break-001.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Test: white-space:break-spaces + word-break:keep-all</title>
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#valdef-white-space-collapse-break-spaces">
+<link rel="help" href="https://drafts.csswg.org/css-text-4/#valdef-word-break-keep-all">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht">
+<meta name="assert" content="break-spaces + keep-all do allow a break after space.">
```

Comment:
```suggestion
<meta name="assert" content="break-spaces + keep-all does allow a break after space.">
```

This is a nit obviously, but it made me consider something I never have before. The result of an expression with "plus" is singular and the result of an "and" expression is plural.

Reply:
Makes sense. I just copied this from another test.